platforms/evm/src/platform.ts

```diff
@@ -43,7 +36,18 @@ export module EvmPlatform {
 
   let contracts: EvmContracts = new EvmContracts(conf);
 
-  type P = typeof platform;
+  export type Type = typeof platform;
+
+  export const {
+    nativeTokenId,
+    isNativeTokenId,
+    isSupportedChain,
+    getDecimals,
+    getBalance,
+    sendWait,
+    getCurrentBlock,
+    chainFromRpc,
+  } = EvmUtils;
```

Comment:
Do you know how to just re-export all of these? Couldn't figure out the right syntax :\